backend/src/apiserver/server/pipeline_server_test.go

```diff
@@ -243,22 +264,9 @@ func TestCreatePipelineV1_ExistingPipeline(t *testing.T) {
 			},
 		},
 	)
-	assert.Nil(t, err)
-	assert.NotEqual(t, "", createdPipeline.GetId())
-	assert.Equal(t, existingPipeline.GetId(), createdPipeline.GetId())
-	assert.Less(t, 0, int(createdPipeline.GetCreatedAt().Seconds))
-
-	newPipelineVersion, err := resourceManager.GetLatestPipelineVersion(createdPipeline.Id)
-	assert.Nil(t, err)
-	assert.NotNil(t, newPipelineVersion)
-	assert.Equal(t, newPipelineVersion.PipelineId, existingPipeline.GetId())
-	assert.Equal(t, newPipelineVersion.PipelineId, createdPipeline.GetId())
-
-	assert.Equal(t, "", existingPipeline.GetDefaultVersion().GetId())
-	assert.Equal(t, 0, int(existingPipeline.GetDefaultVersion().GetCreatedAt().Seconds))
-
-	assert.Equal(t, newPipelineVersion.UUID, createdPipeline.GetDefaultVersion().GetId())
-	assert.Less(t, 0, int(createdPipeline.GetDefaultVersion().GetCreatedAt().Seconds))
+	assert.NotNil(t, err)
```

Comment:
From the result of the test, this looks like a bebavior change with `CreatePipelineV1`. I understand the new behavior may make more sense, but I'm not sure if we should change how v1 works at this point.

Reply:
Agree on the behavior change. I will try to avoid it in the next iteration, but it might be infeasible due to the difference in `Pipeline` across v1 and v2. Besides, some of these changes were introduced due to refactoring and API changes.